src/openlcb/TractionTrain.cxx

```diff
@@ -42,14 +42,18 @@
 namespace openlcb
 {
 
-TrainNode::TrainNode(TrainService *service, TrainImpl *train)
+DefaultTrainNode::DefaultTrainNode(TrainService *service, TrainImpl *train)
     : service_(service)
     , train_(train)
     , isInitialized_(0)
     , controllerNodeId_({0, 0})
 {
 }
 
+DefaultTrainNode::~DefaultTrainNode()
+{
+}
+
 TrainNode::~TrainNode()
```

Comment:
I moved the consisting storage implementation into a new intermediate class, but I did not rename TrainNode to TrainNodeInterface because that class naming convention is not used anywhere in the openmrn codebase.